src/dotnet/CommandLine/CommandLineApplication.cs

```diff
@@ -372,11 +372,11 @@ public void ShowHelp(string commandName = null)
                 if (cmd != this && cmd.Arguments.Any())
                 {
                     var args = string.Join(" ", cmd.Arguments.Select(arg => arg.Name));
-                    headerBuilder.Insert(usagePrefixLength, string.Format(LocalizableStrings.UsageItemWithArgs, cmd.Name, args));
+                    headerBuilder.Insert(usagePrefixLength, string.Format(" {0} {1}", cmd.Name, args));
```

Comment:
I haven't seen this approach used before. Translators typically do the ordering reversals by swapping the token positions within the larger, translated string. A loc string that's nothing but tokens is very hard to understand and the extra indirection will lead to bugs in specific languages that the dev team is unlikely to catch.